### DISCUSSION TITLE
테스트 도중 발견된 버그들

### DISCUSSION THREAD

**Discussion by seong0929** [Tiny-Metro/Poly-Metro#320] [General]
- 같은 역에 클릭을 두 번 할 경우 노선이 해당 역에만 생성.
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/e6a26b2e-e6ee-4aef-8229-579013853e4c)
- 아래와 같은 경우도 순환선으로 판정이 됨(투자 미션 1001)
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/619e832b-cb59-4d85-a4a5-44a38c188bc5)
- 새로운 역이 생성되었음을 알려주는 펄스가 초기 3역에서 펄스가 끊기지 않음
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/deea817d-2593-4926-ae1c-ceb6027c8abf)
- 투자 미션 수락 후(Mon), 미션을 수행하지 않았는 데 3일 후(Thu) 코딩되지 않은 돈(5000)이 들어옴.
   - 현재까지 확인된 투자 미션 1003, 1004, 1005, 1007 ('어떤 역도 불만도 50을 넘지 않도록 하세요.', '을 업그레이드하세요.')
   - 다른 돈이 들어간 보상의 합은 6200 (투자 미션 1001, 1002, 1003, 1004, 1007, 1008, 1009, 1010, 1011)(300, 500, 800, 300, (300, 500), 500, 800, 2000, 500, 500)
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/b086cafe-ae0d-45cd-ab8e-f7b97f2cc35e)
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/7c4089b2-a523-477d-af32-dc200d771938)
    - 앞선 상황 이후 2주차에 진입하여(Mon) 두 미션 수락한 뒤 3일 뒤(Thu)에 미션이 완료됨
    - 화요일(Tue)에 미션 수락 시 금요일(Fri)에 미션이 완료됨
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/4c42baa4-8374-4afc-9196-3baa76c84f37)

- 완료한 미션이 몇일이 지나도 남아 있음
   -  다음주가 되면 사라짐
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/48a9737c-8b50-4569-bc15-61381da0e2bb)

- 대출을 하지 않았는 데, Investment.1009가 등장
<pre>
<code>
function Appearance()
    totalloanmoney = loan.TotalUsingLoanCount
    return totalloanmoney ~= 0
end
</code>
</pre>

**Comment by AzureSoda** [comment]
- 게임플레이 화면에서 햄버거 아이콘에서 to Menu버튼이 눌리지 않음
- 일시정지 버튼을 눌러도 시간이 멈추지 않음
- 게임 플레이를 중단하고 다시 시작하면 카메라가 이전에 마지막 위치에서 시작함
- 게임 플레이를 중단하고 다시 시작하면 이전에 생성된 역과 승객들이 그대로 시작함, 돈은 0부터 시작함
- 게임을 종료하는 방법이 필요함
### 모바일
- Bank,Policy,Shop에서 스크롤을 하려고 하면 카메라 위치가 이동됨
- Bank에서의 글씨가 깨져보임
- 모바일 환경에서는 글씨가 작게보임
### PC
Bank,Policy,Shop에서 스크롤을 하려고 하면 카메라가 줌인,줌아웃됨

**Comment by seong0929** [comment]
이슈화
